financial-connections/src/main/java/com/stripe/android/financialconnections/repository/AttachedPaymentAccountRepository.kt

```diff
@@ -0,0 +1,31 @@
+package com.stripe.android.financialconnections.repository
+
+import android.os.Parcelable
+import androidx.lifecycle.SavedStateHandle
+import com.stripe.android.core.Logger
+import com.stripe.android.financialconnections.model.PaymentAccountParams
+import kotlinx.parcelize.Parcelize
+import javax.inject.Inject
+import javax.inject.Singleton
+
+/**
+ * Repository for attached payment accounts.
+ */
+@Singleton
+internal class AttachedPaymentAccountRepository @Inject constructor(
+    savedStateHandle: SavedStateHandle,
+    private val logger: Logger,
+) : PersistingRepository<AttachedPaymentAccountRepository.State>(
+    savedStateHandle = savedStateHandle,
+) {
```

Comment:
This is used to remember previously attached accounts locally. If an account of type `bank_account` (manually entered) was attached, we'll call `save_accounts_to_link` with `null` as `accounts`.